eZ/Bundle/EzPublishCoreBundle/Tests/Imagine/ImageAsset/AliasGeneratorTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\Tests\Imagine\ImageAsset;
+
+use eZ\Bundle\EzPublishCoreBundle\Imagine\ImageAsset\AliasGenerator;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\FieldType\ImageAsset;
+use eZ\Publish\Core\FieldType\Image;
+use eZ\Publish\Core\Repository\Values\Content\Content;
+use eZ\Publish\Core\Repository\Values\Content\VersionInfo;
+use eZ\Publish\SPI\Variation\VariationHandler;
+use PHPUnit\Framework\TestCase;
+
+class AliasGeneratorTest extends TestCase
+{
+    /**
+     * @var \eZ\Bundle\EzPublishCoreBundle\Imagine\ImageAsset\AliasGenerator
```

Comment:
Should we use a single line or multiline for /** @var .... */ In this PR both are used